lib/Futures/SharedState.h

```diff
@@ -210,6 +211,7 @@ class SharedState {
         [[fallthrough]];
 
       default:
+        break;
         TRI_ASSERT(false);  // unexpected state
```

Comment:
Is the break on purpose before the TRI_ASSERT - then you can just delete the TRI_ASSERT I guess.